apps/style/common.scss

```diff
@@ -293,6 +293,10 @@ $dpad-red: #F78183;
   }
 }
 
+.contained-level-container {
+  min-height: 200px;
+}
```

Comment:
Should be easy to change. Is that now preferred vs. keeping these in CSS classes?